### DISCUSSION TITLE
Electricity scenario LCA

### DISCUSSION THREAD

**Discussion by larettaok** [LCA-ActivityBrowser/activity-browser#1253] [Q&A]
Hey guys I hope in a quick response :( @marc-vdm possible from you that you were so insightful.
I would like to do an energy scenario in the activity browser changing the electricity mix. So i want to open the electricity mix of Switzerland and change the % coming from renewable and fossil/nuclear. How can I do it since i cannot edit the values?

Please save me, thanks :)
<img width="1315" alt="Screenshot 2024-03-14 at 2 29 01 AM" src="https://github.com/LCA-ActivityBrowser/activity-browser/assets/154847702/c4a27d57-ffea-444f-b857-e5ea9049549a">

**Comment by marc-vdm** [comment]
Hey, thanks for reaching out.

First about your other post: We're a group of -almost exclusively- volunteers working on AB in our free time. We try to respond to questions like this when we have time, but please understand we won't always be able to respond quickly. I've closed the discussion there.

There are 2 options to what you're asking:
1. Manually changing the database once with new values
2. Using scenarios to edit the electricity mix

### Manually changing the database
The database you have open seems to be `Read-only` In the screenshot below I've marked the buttons column in purple. Once you enable writing to the database, you can edit activities within a database.
![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/34626062/fb6af530-467a-453e-8087-e45ad3accb5b)

### Using scenarios
You can also use scenarios, for which we have a tutorial [here](https://www.youtube.com/watch?v=3LPcpV1G_jg)


___
One comment of the above options: It seems the database you have open is ecoinvent, I would strongly suggest _never_ manually changing ecoinvent databases yourself, as it will become very hard to track all your changes and other people may not be able to replicate your work as your ecoinvent database the database that ecoinvent gave you anymore. With scenarios, you don't actually write to the database.
